uniswap/uniswap.py

```diff
@@ -1209,6 +1215,51 @@ def get_weth_address(self) -> ChecksumAddress:
             address = self.router.functions.WETH9().call()
         return address
 
+    def get_raw_price(
```

Comment:
New public method which is not supported on v1, so should be annotated with `@supports`.

A short docstring would be nice too :)

Reply:
Well, I've fixed that stuff. Everything seems correct.

Reply:
@ErikBjare ^^